install/kubernetes/cilium/templates/hubble-cli.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.hubble.cli.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hubble-cli
+  labels:
+    k8s-app: hubble-cli
+  namespace: {{ .Release.Namespace }}
+spec:
+  containers:
+  - name: hubble-cli
+    image: quay.io/cilium/hubble:latest
```

Comment:
I think we should pin a version in the `install/kubernetes/Makefile` (similar to how we do it for hubble-ui).